src/visbook/visbook.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { VisbookApi } from './visbook.api';
+
+@Injectable()
+export class VisbookService {
+  private readonly apiClient = new VisbookApi();
+
+  constructor(private readonly configService: ConfigService) {}
+
+  async onModuleInit() {}
```

Comment:
I think this module will ever need to do anything in an on init call so could drop this.